src/drizzle/schema/transactions.ts

```diff
@@ -24,6 +24,7 @@ export const TransactionTable = pgTable("transactions", {
     .references(() => CategoryTable.id, { onDelete: "cascade" }),
   price: integer().notNull(),
   date: timestamp().notNull(),
+  currentBalance: integer().notNull(),
```

Comment:
tu bym poszedł raczej w kierunki `balanceAfterTransaction` albo `balanceBeforeTransaction` żeby nazwa jasno mówiła jakie to saldo